### PR TITLE
Update OpenShift Service Mesh Workshop

### PR DESCRIPTION
**Pull request by theckang**
Updates:
- Added new instructions for instructors and solo participants during install and setup
- Added Summary sections for some labs

### CODE REVIEW

File: content/workshops/openshift_service_mesh/lab1.3_deploymsa.md [hunk 1]
```diff
@@ -58,10 +57,66 @@ spec:
...
+
+Next we need a local copy of our application code.
+
+<blockquote>
+<i class="fa fa-terminal"></i> Clone the repository in your home directory:
+</blockquote>
+
+```
+cd $HOME
+git clone https://github.com/dudash/openshift-microservices.git
```
Comment (dudash) [inline content/workshops/openshift_service_mesh/lab1.3_deploymsa.md]: they only need to clone in solo? Shouldn't this be required for both?

Comment (theckang) [reply content/workshops/openshift_service_mesh/lab1.3_deploymsa.md]: If they're running it on their own, they clone the repo in the previous lab and install Istio using resources from the repo.  I'll add some steps to make sure they check they have the repo installed.

Comment (dudash) [reply content/workshops/openshift_service_mesh/lab1.3_deploymsa.md]: OK, that makes sense. No need to add checks. Can you add the collapsable tags though?

**Comment by dudash** [comment]
oh cool. I see. lgtm

**Pull request merged**